confgenerator/testdata/valid/windows/default_config/golden_otel.conf

```diff
@@ -17,6 +17,35 @@ receivers:
       network:
       paging:
       process:
+  windowsperfcounters/mssql_mssql:
+    collection_interval: 60s
+    perfcounters:
+      - object: SQLServer:General Statistics
+        instances: _Total
+        counters:
+          - User Connections
+      - object: SQLServer:Databases
+        instances: _Total
+        counters:
+          - Transactions/sec
+          - Write Transactions/sec
+  windowsperfcounters/iis_iis:
+    collection_interval: 60s
+    perfcounters:
+      - object: Web Service
+        instances: _Total
+        counters:
+          - Current Connections
+          - Total Bytes Received
+          - Total Bytes Sent
+          - Total Connection Attempts (all instances)
+          - Total Delete Requests
+          - Total Get Requests
+          - Total Head Requests
+          - Total Options Requests
+          - Total Post Requests
+          - Total Put Requests
+          - Total Trace Requests
```

Comment:
This test case appears to be out of sync with `confgenerator/windows-default-config.yaml`. I added this to match what the test case expects.